CMakeLists.txt

```diff
@@ -20,6 +20,15 @@ set_property(TARGET boost PROPERTY
 set_property(TARGET boost PROPERTY
     INTERFACE_LINK_LIBRARIES ${Boost_LIBRARIES})
 
+# Include Folly as an imported target
+find_package(Folly 2017.07.31.00 REQUIRED)
```

Comment:
It turns out that this version pinning was completely ineffective because `FindFolly.cmake` doesn't set any version flags, nor does it appear that folly specifies an installation version in its include directory. For now, I've removed the version entirely.